Marlin/configuration_store.cpp

```diff
@@ -1098,7 +1098,7 @@ void Config_ResetDefault() {
     #endif
     SERIAL_EOL;
     #if ENABLED(DISTINCT_E_FACTORS)
-      for (uint8_t i = 0; i < E_STEPPERS; i++) {
+      for (uint8_t i = 0; i < EXTRUDERS; i++) {
```

Comment:
These should be left as `E_STEPPERS`.

Here we have to be cautious. The `axis_steps_per_mm` array length (`XYZE_N`) is based on `E_STEPPERS`, and these speed and acceleration factors do pertain to the steppers, not to the "extruders." The `DISTINCT_E_FACTORS` configuration option is what determines whether all steppers share the same steps-per-unit (etc.) or whether they each have their own distinct value. When they do have their own values, the `active_extruder` is used to index into these arrays.